sample-error-response-v2.yaml

```diff
@@ -0,0 +1,72 @@
+openapi: '3.0.0'
+info:
+  description: This specification is only for sample error response for API v2.
+  version: '1.0'
+  title: Sample Error Response
+servers:
+  - url: 'http://sample-service'
+paths:
+  /v2/sample/error:
+    get:
+      summary: Sample Request for Error Response
+      responses:
+        500:
+          $ref: '#/components/responses/InternalServerError'
+
+components:
+  responses:
+    InternalServerError:
+      description: Server error. This should happen rarerly.
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/ErrorResponse'
+          example:
+            timestamp: 2022-06-17T14:07:17Z
+            trace_id: aswo3f-a202lfso-312123sld-1230dd1
+            errors:
+              - title: Internal error.
```

Comment:
Suggestion: Since the component is called `InternalServerError`, maybe we could also change the `title` to `Internal Server Error` and `type` to `https://<URL>/errors#sample_service.internal_server_error` as well?